wagtail/admin/templates/wagtailadmin/base.html

```diff
@@ -8,11 +8,14 @@
                 {% block branding_logo %}
                     {# Mobile-only logo: #}
                     <div class="wagtail-logo-container__mobile u-hidden@sm">
-                        <img class="wagtail-logo wagtail-logo__full" src="{% static 'wagtailadmin/images/wagtail-logo.svg' %}" alt="Wagtail" width="80" />
+                        <img class="wagtail-logo-container__background" src="{% static 'wagtailadmin/images/logo-background.svg' %}" alt=""/>
+                        <div class="wagtail-logo-container-inner">
+                          <img class="wagtail-logo wagtail-logo__full" src="{% static 'wagtailadmin/images/wagtail-logo.svg' %}" alt="Wagtail" width="80" />
+                        </div>
                     </div>
 
                     {# Desktop logo (animated): #}
-                    {% include "wagtailadmin/shared/animated_logo.html" %}
+                    {% include "wagtailadmin/shared/animated_logo_background.html" %}
```

Comment:
Could we use a partial that takes a parameter (`with background=True`), or only have the one partial and add the background around it? The two variants are too similar to justify the duplication in two separate files I think.